cmd/skopeo/sync.go

```diff
@@ -121,6 +123,7 @@ See skopeo-sync(1) for details.
 	flags.StringVarP(&opts.destination, "dest", "d", "", "DESTINATION transport type")
 	flags.BoolVar(&opts.scoped, "scoped", false, "Images at DESTINATION are prefix using the full source image path as scope")
 	flags.StringVar(&opts.appendSuffix, "append-suffix", "", "String to append to DESTINATION tags")
+	flags.StringVar(&opts.digestFile, "digestfile", "", "Write the digest of the pushed image to the specified file")
```

Comment:
[I was thinking about naming the option `digest-file`, this is just _ugly_.

But `skopeo copy`, as well as Buildah and Podman, all use the dash-less variant. And we have `--authfile`.

So I guess `digestfile` makes the most sense; still, highlighting this as a possible point of discussion.]